### PR TITLE
FIX: installing winetricks verbs early

### PR DESCRIPTION
**Pull request by mmtrt**
Since some installers required verbs in wineprefix to work properly.

This also fixes TRICKS envar to work properly otherwise had to add verbs manually to pre-install script to fix installer requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ See the following snaps for complete examples of how to use sommelier-core.
 
 * [PhotoScape snap](https://github.com/snapcrafters/photoscape)
 * [Bridge Designer snap](https://github.com/snapcrafters/bridge-designer)
-* [TrackMania Nations Forever snap](https://github.com/galgalesh/tmnationsforever)
+* [TrackMania Nations Forever snap](https://github.com/snapcrafters/tmnationsforever)
```
Comment (lucyllewy) [inline README.md:83]: Please revert this change - it is irrelevant to the PR. If you want to change the URL then that should be a separate PR.

**Review by lucyllewy — CHANGES_REQUESTED** [review]

**Comment by merlijn-sebrechts** [comment]
The edit Dani is talking about comes from a commit on `master` that hasn't been merged to `1.0` yet. We discussed this privately and decided for a better way to do releases. From now on, we'll merge the `master` branch into `1.0` ourselves, to avoid future conflicts. I'm closing this PR and will create a new one myself. Thanks for your help, @mmtrt !
